src/components/FeeExplainer/FeeSliders.tsx

```diff
@@ -100,9 +144,21 @@ export const FeeSliders: React.FC<FeeSlidersProps> = ({
         </Stack>
       </Stack>
       <Stack width='full' spacing={4}>
-        <Flex width='full' justifyContent='space-between' fontWeight='medium'>
+        <Flex width='full' justifyContent='space-between' alignItems='center' fontWeight='medium'>
           <Text translation={featureSizeTranslation} />
-          <Amount.Fiat fiatType='USD' value={tradeSizeUSD} fontWeight='bold' />
+          <Box sx={inputStyle} width='35%'>
+            <NumberFormat
+              decimalScale={2}
+              customInput={Input}
+              isNumericString={true}
+              prefix={'$'}
+              decimalSeparator={'.'}
+              inputMode='decimal'
+              thousandSeparator={','}
```

Comment:
Same here

Reply:
Changed in [6b12ae3](https://github.com/shapeshift/web/pull/7984/commits/6b12ae3b57f16c7a37fc2699a84daf2c62b3147f)